connect/api/src/main/java/org/apache/kafka/connect/source/SourceTask.java

```diff
@@ -96,7 +96,7 @@ public void initialize(SourceTaskContext context) {
      * should block but return control to the caller regularly (by returning {@code null}) in
      * order for the task to transition to the {@code PAUSED} state if requested to do so.
      * <p>
-     * The task will be {@link #stop() stopped} on a separate thread, and when that happens
+     * The task may be {@link #stop() stopped} on a separate thread, and when that happens
```

Comment:
See [KAFKA-15090](https://issues.apache.org/jira/browse/KAFKA-15090) for context on this change.